scripts/s3.py

```diff
@@ -41,13 +41,22 @@
 def wrap_vault(
     vault: Union[VaultV1, VaultV2], samples: ApySamples, aliases: dict, icon_url: str, assets_metadata: dict
 ) -> dict:
-    apy = vault.apy(samples)
-
+    apy_error = Apy("error", 0, 0, ApyFees(0, 0), ApyPoints(0, 0, 0))
+    try:
+        apy = vault.apy(samples)
+    except ValueError as error:
```

Comment:
Don't completely fail to update or inject the vault if we cannot properly calculate APY